### PR TITLE
Improve code quality

### PR DESCRIPTION
**Pull request by joaoantoniocardoso**
We might not want the `module_inception` ([link](https://rust-lang.github.io/rust-clippy/master/index.html#module_inception)) fix, @patrickelectric can you check it?

### CODE REVIEW

**Review by patrickelectric — COMMENTED** [review]
Sorry, I forgot to submit the review!

File: build.rs [hunk 1]
```diff
@@ -4,20 +4,20 @@ use std::path::Path;
...
 
 fn file_download(url: &str, output: &str) {
-    let mut resp = reqwest::blocking::get(url).expect(&format!("Failed to download file: {url}"));
+    let mut resp =
+        reqwest::blocking::get(url).unwrap_or_else(|_| panic!("Failed to download file: {url}"));
     let file_path = std::path::Path::new(env!("CARGO_MANIFEST_DIR")).join(output);
-    let mut output_file =
-        std::fs::File::create(&file_path).expect(&format!("Failed to create file: {file_path:?}"));
+    let mut output_file = std::fs::File::create(&file_path)
+        .unwrap_or_else(|_| panic!("Failed to create file: {file_path:?}"));
```
[the remark this answers is not in the document]
Comment (joaoantoniocardoso) [reply build.rs:20]: .expect() is not supposed to be used with a function call as a parameter, that's why it suggests this. The same kind of idea also applies to .then() vs .and_then()

**Comment by patrickelectric** [comment]
@joaoantoniocardoso what is the status ?

**Comment by joaoantoniocardoso** [comment]
> @joaoantoniocardoso what is the status?

We've agreed about the `return` statements at that meeting, so we can merge it

**Review by patrickelectric — APPROVED** [review]

**Pull request merged**